src/clj/rems/application/model.clj

```diff
@@ -572,10 +574,13 @@
 (defn see-application? [application user-id]
   (not= #{:everyone-else} (permissions/user-roles application user-id)))
 
+(defn- hide-commands [commands]
+  (set/difference commands (set (:hide-commands env))))
+
 (defn apply-user-permissions [application user-id]
   (let [see-application? (see-application? application user-id)
         roles (permissions/user-roles application user-id)
-        permissions (permissions/user-permissions application user-id)
+        permissions (hide-commands (permissions/user-permissions application user-id))
```

Comment:
The command could be truly disabled by using `rems.permissions/blacklist` in `rems.application.model/calculate-permissions`

Reply:
done